modules/fmops/sagemaker-hugging-face-endpoint/README.md

```diff
@@ -0,0 +1,60 @@
+# SageMaker Hugging Face Foundation Model Endpoint
+
+## Description
+
+Deploys an endpoint for a foundation model from [Hugging Face](https://huggingface.co/).
```

Comment:
Can we pls clarify "foundation model supported by [Hugging Face LLM Inference Containers for Amazon SageMaker](https://huggingface.co/blog/sagemaker-huggingface-llm)".